network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -284,33 +285,36 @@ <h3 class="h2-heading mb-4 mb-md-5">{% trans "Updates" %}</h3>
 
           <h3 class="h2-heading mb-4">{% trans "Related products" %}</h3>
           <div class="row">
-            {% for product in product.related_products %}
+            {% for related_product_page in product.related_product_pages.all %}
+            {% with related_product=related_product_page.related_product %}
```

Comment:
Getting fancy with `{% with %}` as to reduce page queries.

Reply:
wxcwllwnt.